app/models/tag_group.rb

```diff
@@ -55,7 +55,9 @@ def self.find_id_by_slug(slug)
   def self.resolve_permissions(permissions)
     permissions.map do |group, permission|
       group_id = Group.group_id_from_param(group)
-      permission = TagGroupPermission.permission_types[permission] unless permission.is_a?(Integer)
+      permission = TagGroupPermission.permission_types[permission.to_sym] unless permission.is_a?(
+        Integer,
+      )
```

Comment:
if you want to golf it into a single line again 😂 (I'm not marking it as a suggestion 😄)

```rb
      permission = TagGroupPermission.permission_types[permission] if !(Integer === permission)
```